custom_components/powercalc/translations/pl.json

```diff
@@ -50,6 +53,7 @@
                     "group_power_entities": "Członek encji mocy",
                     "name": "Nazwa",
                     "sub_groups": "Podgrupy",
+					"hide_members": "Ukryj członków",
```

Comment:
Could you fix the indenting please? Probably you used tabs instead of spaces and you have tabstop of 2